code/modules/mob/living/simple_animal/hostile/megafauna/dragon.dm

```diff
@@ -41,6 +42,13 @@
 		return
 	else
 		..()
+		if(isliving(target))
+			var/mob/living/L = target
+			if(L.stat)
+				L.gib()
+				visible_message("<span class='danger'>[src] devours [L]!</span>")
+				src << "<span class='userdanger'>You feast on [L], restoring your health!</span>"
+				L.adjustBruteLoss(-L.maxHealth)
```

Comment:
please good lord no this is a TON of healing and normal ash drakes will do it too